Quantum-Control-Applications/ESR/signal_test.py

```diff
@@ -0,0 +1,147 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+###################
+# The QUA program #
+###################
+
+pulse1_len = 400 // 4
+pulse2_len = pulse1_len * 2
+pulse_delay = 2000 // 4 - pulse1_len / 2 - pulse2_len / 2
+readout_delay = 2000 // 4 - pulse2_len / 2 - readout_len // 8 - 8
+
+cooldown_time = int(10e6 // 4)
+
+n_avg = 100
+
+with program() as signal_test:
+
+    n = declare(int)
+    n_st = declare_stream()
+
+    with for_(n, 0, n < n_avg, n + 1):
+
+        # wait(cooldown_time)
+
+        reset_phase("ensemble")
+        reset_phase("resonator")
+        reset_frame("ensemble")
+
+        echo = declare_stream(adc_trace=True)
+
+        wait(8, "switch_1", "switch_2")
```

Comment:
Why do you sometimes wait 8 cc or 11 cc ?

Reply:
added docstrings